WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -23,15 +23,15 @@ struct TotalsView: View {
                             VStack(spacing: Constants.totalsVerticalSpacing) {
                                 priceFieldView(title: "Subtotal",
```

Comment:
Non-blocker: Since we're editing this bit, we can move the title for "Subtotal" and "Taxes" titles to an `NSLocalizedString`